int32 id
# the aruco id
float32[] corners
# length(corners) is eight for 1.x, 1.y, 2.x, 2.y, 3.x ,3.y, 4.x, 4.y